server/sonar-web/src/main/js/app/utils/installExtensionsHandler.js

```diff
@@ -20,14 +20,14 @@
 // @flow
 const extensions = {};
 
-const registerExtension = (key: string, start: Function) => {
+const registerExtension = (key /*: string */, start /*: Function */) => {
   extensions[key] = start;
 };
 
 export default () => {
   window.registerExtension = registerExtension;
 };
 
-export const getExtensionFromCache = (key: string) => {
+export const getExtensionFromCache = (key /*: string */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove curly braces and "return" from this arrow function body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)